packages/sanity/src/core/create/components/DialogPortalProvider.tsx

```diff
@@ -0,0 +1,10 @@
+import {PortalProvider, usePortal} from '@sanity/ui'
+import {type ReactNode} from 'react'
+
+export function DialogPortalProvider(props: {portalElementId: string; children: ReactNode}) {
```

Comment:
Huh, could have sworn we had already something like this? 🤔 
Somewhere in the studio? Maybe I'm just misremembering and what I remember is from your previous work 😅

Reply:
@RitaDias You may be remembering this component from a previous iteration. The only change is that this is now exported as it's used within multiple components (in this integration at least)